utkdigitalinitiatives/utk_isl_xml_forms
# Address TRAC-378 TRAC-397

**JIRA Ticket**: [TRAC-378](https://jira.lib.utk.edu/browse/TRAC-378) and [TRAC-397](https://jira.lib.utk.edu/browse/TRAC-397)

# What does this Pull Request do?
This PR adds a post_process_transforms directory with three separate transforms. This PR also updates TRAC-397.

# What's new?
Three transforms that should strip empty elements from the MODS datastream.

# How should this be tested?
Copy the UTK_ir_etds_post_process.xsl transform to `sites/all/modules/islandora_xml_forms/builder/self_tranforms/` and, using the association process, link the XSL transform with an appropriate form. Check the output in the MODS datastream.

# Interested parties
@markpbaggett
